docs/README.md

```diff
@@ -24,6 +24,16 @@ Theme is copied from Daml Docs theme.
 
 `Node` and `yarn` must be installed on your machine.
 
+### Sass
+
+The `Node` version of `sass` must be installed: `gem install sass`
+
+### Grunt
+
+If grunt is not already installed: `sudo npm install -g grunt-cli`
+
+If you had to install anything new above, you may have to recreate the virtual environment: `pipenv install` again
```

Comment:
```suggestion
If you had to install anything new above, you may have to recreate the virtual environment: `pipenv install` again.
```

Reply:
I am not sure this comment is needed, unless we have to change the repo structure again.

On a standard setup it should work out of the box.

Reply:
I added an extra comment, to only try it if it does not work despite installing the required libraries.